docs/pages/versions/unversioned/guides/using-custom-fonts.md

```diff
@@ -2,124 +2,160 @@
 title: Using Custom Fonts
 ---
 
-Both iOS and Android come with their own set of platform fonts but if you want to inject some more brand personality into your app, a well picked font can go a long way. In this guide we'll walk you through adding a custom font to your Expo app. We'll use [Open Sans](https://fonts.google.com/specimen/Open+Sans) from [Google Fonts](https://fonts.google.com/) in the example, and the process is identical for any other font, so feel free to adapt it to your use case. Before proceeding, go ahead and download [Open Sans](https://fonts.google.com/specimen/Open+Sans)
+import SnackEmbed from '~/components/plugins/SnackEmbed';
 
-## Starting code
+Both iOS and Android and most desktop operating systems come with their own set of platform fonts but if you want to inject some more brand personality into your app, a well picked font can go a long way. And since each
+operating system has its own set of platform fonts, if you want to produce an experience that is consistent for all users, you'll want to use your own fonts in your project. This guide will show you how to do that.
 
-First let's start with a basic "Hello world!" app. Create a new project with `expo init` and change `App.js` to the following:
+## A Minimal but Complete Working Example
 
-```javascript
-import React from 'react';
-import {
-  Text,
-  View,
-} from 'react-native';
-
-export default class App extends React.Component {
-  render() {
-    return (
-      <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-        <Text style={{ fontSize: 56 }}>
-          Hello, world!
-        </Text>
-      </View>
-    );
-  }
-}
-```
+<SnackEmbed snackId="@ccheever/custom-font-example" />
```

Comment:
ideally we want to be able to embed snacks inline because they are versioned properly and anyone can send a pr to fix.  see an example of this here: https://github.com/expo/expo/blame/master/docs/pages/versions/v36.0.0/tutorial/text.md#L14

that said, it's not currently possible to include assets in an inline snack. it may be a good project for you if you're interested @ccheever to add the capability to specify an asset to download and include in the project in a query param

Reply:
Will take a look at this in the future; seems like a good idea and I def prefer inline snacks, esp. for docs.